src/pke/unittest/UnitTestMultiparty.cpp

```diff
@@ -186,73 +186,137 @@ static std::vector<TEST_CASE_UTGENERAL_MULTIPARTY> testCases = {
-    { BFVRNS_TEST, "24", {BFVRNS_SCHEME,   DFLT, DFLT,      60,       20,    DFLT,    GAUSSIAN,        DFLT,          DFLT,     DFLT,         DFLT,   DFLT,         DFLT,    65537, DFLT,   DFLT,      DFLT, HPSPOVERQLEVELED, EXTENDED, DFLT},   false,    0},
-    { BFVRNS_TEST, "25", {BFVRNS_SCHEME,   DFLT, DFLT,      60,       20,    DFLT,    UNIFORM_TERNARY, DFLT,          DFLT,     DFLT,         DFLT,   DFLT,         DFLT,    65537, DFLT,   DFLT,      DFLT, HPS,              EXTENDED, DFLT},   true,     0},
-    { BFVRNS_TEST, "26", {BFVRNS_SCHEME,   DFLT, DFLT,      60,       20,    DFLT,    GAUSSIAN,        DFLT,          DFLT,     DFLT,         DFLT,   DFLT,         DFLT,    65537, DFLT,   DFLT,      DFLT, HPS,              EXTENDED, DFLT},   true,     0},
-    { BFVRNS_TEST, "27", {BFVRNS_SCHEME,   DFLT, DFLT,      60,       20,    DFLT,    UNIFORM_TERNARY, DFLT,          DFLT,     DFLT,         DFLT,   DFLT,         DFLT,    65537, DFLT,   DFLT,      DFLT, BEHZ,             EXTENDED, DFLT},   true,     0},
-    { BFVRNS_TEST, "28", {BFVRNS_SCHEME,   DFLT, DFLT,      60,       20,    DFLT,    GAUSSIAN,        DFLT,          DFLT,     DFLT,         DFLT,   DFLT,         DFLT,    65537, DFLT,   DFLT,      DFLT, BEHZ,             EXTENDED, DFLT},   true,     0},
-    { BFVRNS_TEST, "29", {BFVRNS_SCHEME,   DFLT, DFLT,      60,       20,    DFLT,    UNIFORM_TERNARY, DFLT,          DFLT,     DFLT,         DFLT,   DFLT,         DFLT,    65537, DFLT,   DFLT,      DFLT, HPSPOVERQ,        EXTENDED, DFLT},   true,     0},
-    { BFVRNS_TEST, "30", {BFVRNS_SCHEME,   DFLT, DFLT,      60,       20,    DFLT,    GAUSSIAN,        DFLT,          DFLT,     DFLT,         DFLT,   DFLT,         DFLT,    65537, DFLT,   DFLT,      DFLT, HPSPOVERQ,        EXTENDED, DFLT},   true,     0},
-    { BFVRNS_TEST, "31", {BFVRNS_SCHEME,   DFLT, DFLT,      60,       20,    DFLT,    UNIFORM_TERNARY, DFLT,          DFLT,     DFLT,         DFLT,   DFLT,         DFLT,    65537, DFLT,   DFLT,      DFLT, HPSPOVERQLEVELED, EXTENDED, DFLT},   true,     0},
-    { BFVRNS_TEST, "32", {BFVRNS_SCHEME,   DFLT, DFLT,      60,       20,    DFLT,    GAUSSIAN,        DFLT,          DFLT,     DFLT,         DFLT,   DFLT,         DFLT,    65537, DFLT,   DFLT,      DFLT, HPSPOVERQLEVELED, EXTENDED, DFLT},   true,     0},
+    // TestType,   Descr, Scheme,          RDim, MultDepth, SModSize, DSize, BatchSz, SecKeyDist,      MaxRelinSkDeg, FModSize, SecLvl,       KSTech, ScalTech,     LDigits, PtMod, StdDev, EvalAddCt, KSCt, MultTech,         EncTech,  PREMode, MultipartyMode,             Star, Slots
+    { BFVRNS_TEST, "01", {BFVRNS_SCHEME,   DFLT, DFLT,      60,       20,    DFLT,    UNIFORM_TERNARY, DFLT,          DFLT,     DFLT,         DFLT,   DFLT,         DFLT,    65537, DFLT,   DFLT,      DFLT, HPS,              STANDARD, DFLT,    FIXED_NOISE_MULTIPARTY},    false,    0},
```

Comment:
just a note: you could use DFLT instead of FIXED_NOISE_MULTIPARTY

Reply:
I think it's clearer in this file to use FIXED_NOISE_MULTIPARTY. For example, when we are specifically testing scaling techniques, we label them FIXEDMANUAL, FIXEDAUTO, FLEXIBLEAUTO, FLEXIBLEAUTOEXT. We don't label one of them as DFLT.